lightly/api/utils.py

```diff
@@ -196,7 +196,7 @@ def get_api_client_configuration(
     token: Optional[str] = None,
     raise_if_no_token_specified: bool = True,
 ) -> Configuration:
-    host = getenv("LIGHTLY_SERVER_LOCATION", "https://api.lightly.ai")
+    host = getenv("LIGHTLY_SERVER_LOCATION", "https://api.lightly.ai").rstrip("/")
```

Comment:
maybe also add a `trim().rstrip("/")` to also safeguard for people adding space at the end by accident? just a thought

Reply:
Sure, that seems like a weirder mistake to make (it's often not easy and it's always obviously wrong to include whitespace in env variables!), but we might as well prevent that here too.